code/game/objects/items/weapons/melee/melee_misc.dm

```diff
@@ -79,6 +79,129 @@
 	plastitanium edge that can cause untold harm to a soft target. In the right hands, it can be a terrifying weapon to behold, \
 	and it’s said that blood runs down the blade in just the right way, to drip artfully from the twin ‘fangs’ at its apex."
 
+// Unathi Sword
+/obj/item/melee/breach_cleaver
+	name = "breach cleaver"
+	desc = "Massive, heavy, and utterly impractical. This sharpened chunk of steel is too big and too heavy to be called a sword."
+	lefthand_file = 'icons/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons_righthand.dmi'
+	base_icon_state = "breach_cleaver"
+	icon_state = "breach_cleaver0"
+	flags = CONDUCT
+	force = 5
+	throwforce = 5
+	w_class = WEIGHT_CLASS_BULKY
+	sharp = TRUE
+	origin_tech = "combat=6;syndicate=5"
+	attack_verb = list("slashed", "cleaved", "chopped")
+	hitsound = 'sound/weapons/swordhitheavy.ogg'
+	materials = list(MAT_METAL = 2000)
+	///How much damage the sword does when wielded
+	var/force_wield = 40
+
+/obj/item/melee/breach_cleaver/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/two_handed, force_wielded = force_wield, force_unwielded = force, icon_wielded = "[base_icon_state]1", wield_callback = CALLBACK(src, PROC_REF(wield)), unwield_callback = CALLBACK(src, PROC_REF(unwield)))
+	AddComponent(/datum/component/parry, _stamina_constant = 2, _stamina_coefficient = 0.5, _parryable_attack_types = NON_PROJECTILE_ATTACKS)
+
+/obj/item/melee/breach_cleaver/examine(mob/user)
+	. = ..()
+	if(isAntag(user))
+		. += "<span>When wielded, this blade has different effects depending on your intent, similar to a martial art. \
+			Help intent will strike with the flat, dealing stamina, disarm intent forces them away, grab intent knocks down the target, \
+			and harm intent deals heavy damage.</span>"
```

Comment:
Were you intending to attach a class to this `<span>` tag here?

Reply:
I was, but it slipped my mind. This should be a notice class, to tint this blue to denote that antagonists know this bit in the description. I've made that change now.